en: update handle-alerts for newly added panels (#271)
pingcap/docs-dm: en/handle-alerts.md

This document introduces how to deal with the alert information in DM.

> ## High availability alert

Reviewer: ```suggestion
## Alerts related to high availability

```

### `DM_worker_offline`

- Description:

    If a DM-worker node is offline for more than one hour, this alert is triggered. In a high-availability architecture, this alert might not directly interrupt the task but increases the risk of interruption.

- Solution:

    You can take the following steps to handle the alert:

    1. View the working status of the corresponding DM-worker node.
    2. Check whether the node is connected.

>    3. Troubleshoot errors through logs.

Reviewer: ```suggestion
    3. Troubleshoot errors through logs.
```

### `DM_DDL_error`

- Description:

    An error occurs while running shard DDL.

- Solution:

    Refer to [Troubleshoot DM](error-handling.md#troubleshooting).

### `DM_pending_DDL`

- Description:

>     If the shard DDL is pending for more than one hour, an alert is triggered.

Reviewer: ```suggestion
    If a sharding DDL operation is pending for more than one hour, this alert is triggered.
```

- Solution:

>     In some usage cases, the pending shard DDL may be expected. In other cases, refer to [Handle Sharding DDL Locks Manually in DM](feature-manually-handling-sharding-ddl-locks.md).

Reviewer: ```suggestion
    In some scenarios, the pending sharding DDL operation might be what users expect. Otherwise, refer to [Handle Sharding DDL Locks Manually in DM](feature-manually-handling-sharding-ddl-locks.md) for solution.
```
